docker-compose.yml

```diff
@@ -25,17 +25,20 @@ services:
       # TODO: remove this in production
       - PYTHONUNBUFFERED=1
       - RPCDEBUGPORT=${RPCDEBUGPORT}
+      - WEBREQUESTPORT=${WEBREQUESTPORT}
+      - WEBREQUESTHOST=${WEBREQUESTHOST}
+      - WEBREQUESTPROTOCOL=${WEBREQUESTPROTOCOL}
     ports:
       - "${RPCPORT}:${RPCPORT}"
       - "${RPCDEBUGPORT}:${RPCDEBUGPORT}"
     volumes:
       - ./.env:/app/.env
       - ./backend:/app/backend
     depends_on:
-      ollama:
-        condition: service_started
       database-migration:
         condition: service_completed_successfully
+      webrequest:
+        condition: service_started
     expose:
```

Comment:
could you add a healthcheck to the webrequest module?

Reply:
added check for selenium in 9539ce042d8c775ca019017333961e215ead1dfb